new/LIT7300PrPhanuel.xml

```diff
@@ -0,0 +1,69 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7300PrPhanuel" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Salām to Phanuel and prayer for a safe delivery</title>
+                <title xml:lang="gez" xml:id="t2">ሰላም፡ ለልሳንከ፡ ዘኢጸርዕ፡ ስብሐታተ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Salām la-Fānuʾel za-ʾi-ṣarǝʿ sǝbḥatāt</title>
```

Comment:
Salām la-ləssānəka

səbḥatāta

I think it's quite safe to emend the text to read ዘኢይጸርዕ፡ (za-ʾi-yəṣarrəʿ) and translate:

"Salutation to your tongue, which does not tire of praises"